azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -143,6 +175,15 @@ public void run() throws Exception {
     generateProperties(propFiles[1]);
   }
 
+  private boolean determineExecuteAsUser(Props sysProps, Props jobProps) {
+    boolean isExecuteAsUser = sysProps.getBoolean(EXECUTE_AS_USER, false);
+    // putting an override in case user needs to override. A temporary opening
+    if (jobProps.containsKey(EXECUTE_AS_USER_OVERRIDE))
+      isExecuteAsUser = jobProps.getBoolean(EXECUTE_AS_USER_OVERRIDE, false);
```

Comment:
1 You don't need default value of "false" as you have already verified that EXECUTE_AS_USER_OVERRIDE is present in jobProps
2 nit parenthesis ?